--- a/src/electron.renderer/misc/JsTools.hx
+++ b/src/electron.renderer/misc/JsTools.hx
@@ -605,7 +605,9 @@ class JsTools {
 	}
 
 	public static function getLogPath() {
-		return getExeDir()+"/LDtk.log";
+		var logDir = Sys.environment().get("XDG_CACHE_HOME");
+		logDir = if (logDir == null) { Sys.environment().get("HOME")+"/.cache"; } else { logDir; }
+		return logDir+"/LDtk.log";
 	}
 
 	public static function getExeDir() {
@@ -618,8 +620,7 @@ class JsTools {
 	}
 
 	public static function getSamplesDir() {
-		var raw = getExeDir() + ( App.isMac() ? "/../samples" : "/samples" );
-		return dn.FilePath.fromDir( raw ).directory;
+		return dn.FilePath.fromDir( "/usr/share/ldtk/samples" ).directory;
 	}
 
 	public static function getAppResourceDir() {
