diff --git a/src.renderer/page/Home.hx b/src.renderer/page/Home.hx
index b2ef8288..2c93b468 100644
--- a/src.renderer/page/Home.hx
+++ b/src.renderer/page/Home.hx
@@ -218,7 +218,7 @@ class Home extends Page {
 	}
 
 	public function onLoadSamples() {
-		var path = JsTools.getExeDir()+"/samples";
+		var path = "/usr/share/ldtk/samples";
 		dn.electron.Dialogs.open(["."+Const.FILE_EXTENSION], path, function(filePath) {
 			App.ME.loadProject(filePath);
 		});
