diff --git a/src.renderer/App.hx b/src.renderer/App.hx
index e1775448..d1299ee4 100644
--- a/src.renderer/App.hx
+++ b/src.renderer/App.hx
@@ -22,7 +22,8 @@ class App extends dn.Process {
 		super();
 
 		// Init logging
-		LOG.logFilePath = JsTools.getExeDir()+"/LDtk.log";
+		var logPath = Sys.environment().get("XDG_CACHE_HOME");
+		LOG.logFilePath = if (logPath == null) { Sys.environment().get("HOME")+"/.cache/ldtk.log"; } else { logPath+"/ldtk.log"; }
 		LOG.trimFileLines();
 		LOG.emptyEntry();
 		LOG.tagColors.set("tidy", "#8ed1ac");
diff --git a/src.renderer/misc/JsTools.hx b/src.renderer/misc/JsTools.hx
index 3d8c97fb..2dc5cf9f 100644
--- a/src.renderer/misc/JsTools.hx
+++ b/src.renderer/misc/JsTools.hx
@@ -551,7 +551,7 @@ class JsTools {
 	}
 
 	public static function getSamplesDir() {
-		return dn.FilePath.fromDir( getExeDir()+"/samples" ).directory;
+		return dn.FilePath.fromDir( "/usr/share/ldtk/samples" ).directory;
 	}
 
 	public static function getAppResourceDir() {
