diff --git a/src.renderer/page/Home.hx b/src.renderer/page/Home.hx
index 05b16803..c7dca36a 100644
--- a/src.renderer/page/Home.hx
+++ b/src.renderer/page/Home.hx
@@ -222,7 +222,7 @@ class Home extends Page {
 	}
 
 	public function onLoadSamples() {
-		var path = JsTools.getExeDir()+"/samples";
+		var path = "/usr/share/LDtk/samples";
 		dn.electron.Dialogs.open(["."+Const.FILE_EXTENSION], path, function(filePath) {
 			loadProject(filePath);
 		});
